pkg/database/database.go

```diff
@@ -236,39 +265,14 @@ func (d *db) releaseTx(tx *store.Tx) {
 }
 
 func (d *db) initSQLEngine() error {
-	// Warn about existent SQL data
```

Comment:
Why is this change needed?

Reply:
to be able to open the store with ExternalCommitAllowance enabled or disabled, but not requiring to switch to special cases because initialization transactions must be committed before